### DISCUSSION TITLE
Does the nrf-DTM support NCS version 2.3.0

### DISCUSSION THREAD

**Discussion by NancyJayakumar** [olleheugene/nRF-DTM#7] [Q&A]
I'm using nrf52840 custom board, when i connect it with the nrf-DTM. I see some output. But there is no help for the  error or warning messages. Is there any documetation for this. 
Also the firmware changes you have mentioned is the nrf5-SDK, does the nrf-DTM also work with nordic SDK(NCS version 2.3.0?) 
the NCS 2.3.0 doesnt have the ble_dtm.

**Comment by olleheugene** [comment]
Can you share the message you have seen?
Unfortunately, I don't have the code for NCS for now. But I will look at the code whether it is possible to do simply.

**Reply by NancyJayakumar** [reply]
Hi, Please find the screen shot, using the code NCS version 2.3.0. 
The custom board is connected to desktop using UART/USB and this test is run from the nrf-DTM.
<img width="400" alt="image" src="https://github.com/olleheugene/nRF-DTM/assets/135833259/57cdb58a-91c5-4ad7-8a29-24d69e63e04d">

**Reply by olleheugene** [reply]
Hi,
It seems the nRF DTM couldn't communicate with the target device.
There can be multiple reasons.
1. The target device does not support DTM
2. Wrong baudrate setting
3. Wrong COM port

Have you used the baudrate as 19200 bps?
The default baudrate of DTM firmware is 19200.

**Reply by NancyJayakumar** [reply]
The target is supported by the DTM : nrf52840
The baudrate recommended on the DTM, also because of my project need  works on 115200 for the RTT, and we are using the same port<img width="167" alt="image" src="https://github.com/olleheugene/nRF-DTM/assets/135833259/312b1032-2f19-4735-8eca-bf9f3751e3c4">
the Com PORT seems to see the communication from the custom board in nrf serial communication. 
Is there anything else, I'm missing?

**Reply by olleheugene** [reply]
The DTM is using UART which means it does not follow the RTT setting.
Have you tested 19200bps? this baudrate is default and it's recommended.
